### ISSUE TITLE
How to use this plugin in Musescore 4

### ISSUE THREAD

**Issue by kyeon-go** [euwbah/musescore-n-tet-plugins#41]
1. Install Plug-in
2. Open all qml files with any text editor (ex : notepad++)
3. For all files, replace Ms.CHORD to Element.CHORD and Qt.quit() to quit()
4. Save all files

In MS 4, Ms namespace dropped and Qt.quit() now also terminates the program!!
Thanks to @cbjeukendrup
(https://github.com/musescore/MuseScore/issues/14754#issuecomment-1328056560)

Note: Muse Sound doesn't support microtone currently. If your playback is Muse Sound, change it to Basic sound.

**Comment by kyeon-go** [comment]
I replaced Ms.CHORD to Element.CHORD in all qmls, because Ms dropped. But all plugin crash Musescore

**Comment by zhenlige** [comment]
It still does not work, and it seems that Musescore 4 does not support microtonal music at all.

**Comment by Trebor-Huang** [comment]
No, Musescore 4 still supports tuning in cents. I can probably investigate why it doesn't work though, after my exams.

**Comment by Trebor-Huang** [comment]
Also the pitch up/down keys work, but only after you press N (toggle note input) does the effect show. Otherwise everything seems to work fine.

**Comment by Trebor-Huang** [comment]
Ah and also, although the playback when you click on specific notes are correct, the main playback doesn't seem to update unless some other change is made.

**Comment by zhenlige** [comment]
The pitches are rounded to 12edo ones when playing in either way.

**Comment by kyeon-go** [comment]
@zhenlige @Trebor-Huang Currently Muse Sound doesn't support microtone.
Basic sound still supports microtone,
 but  I have to restart musescore after running tune to n-edo.

Since MU4 RC, Muse Sound seems to become default playback

**Comment by Trebor-Huang** [comment]
Ah I see, I have been trying out VST, so I'm using Pianoteq to test all these. So I guess everything makes sense now?

My current workflow is to tune everything, select everything and give them some articulations, and immediately delete the articulations. This seems to set everything right.

**Comment by kyeon-go** [comment]
So, does this plug-in works well now?
I'll try your workflow!

By the way, does Pianoteq support n-edo tuning?

P.S : I wish microtonal support for Muse Sound

**Comment by Trebor-Huang** [comment]
You can enable MIDI tuning in the MIDI settings for Pianoteq, this way it is directly compatible with this plugin. Alternatively you can go to the tuning tab in Pianoteq, and generate say a 19-TET tuning. But this way the MIDI input should be renumbered (B becomes Bb, C becomes B, C# becomes B#, D becomes C etc), this is not compatible with this plugin, but it is possible with some code changes. I wonder if it is worth having.

**Comment by Trebor-Huang** [comment]
Okay I think I figured it out, ~~PR on the way~~ PR #42. See [here](https://musescore.org/en/node/325393), but also [here](https://musescore.org/en/node/337468).
